### DISCUSSION TITLE
Officially maintained Flatpak version

### DISCUSSION THREAD

**Discussion by sith-on-mars** [Zettlr/Zettlr#4446] [General]
I found that the Flatpak version of Zettlr often lagging behind. Updates can take up to weeks to take place. Would there be any chance that Flatpak version can be maintained by the team? Currently the Flatpak version simple repackages the appimage file so it shouldn't be too complex to maintain. Unfortunately I don't know how to write pull requests or do any coding.

Flatpak is being embraced by more people and basically becomes the default in some distros (like almost all immutable distros).

Disclaimer, I really appreciate the efforts put by all of the volunteers.

**Comment by nathanlesage** [comment]
I cannot maintain anything the cannot be done with a GitHub action. So if you could collect a series of exact steps I would need to do in order to publish a release via automation as a flat pack, I can absolutely integrate that into the build pipeline. It might be even better if you could already write a workflow that I can implement and run on every release cycle.

**Comment by sith-on-mars** [comment]
It's perfectly understandable. Thanks a lot for explaining. Hopefully someone knowledgable can chip in and help with this.

**Reply by nathanlesage** [reply]
Quick update because I just realized what our last status was: there wasn't a way to automatically publish to flat hub, which is why a community member offered to take this on them, and I would like to keep it that way for now, because again changing this adds complexity on my plate that I can't really deal with at the moment. You may have more luck asking them directly on the repo, maybe they can explain to you how it works so that you can share the work?
